### DISCUSSION TITLE
Swap ET and BT on Aillio Bullet R1 V2 during roasting

### DISCUSSION THREAD

**Discussion by aqdennis** [artisan-roaster-scope/artisan#980] [Q&A] [answered]
Hi,

We're using an Aillio Bullet R1 V2 for sample roasting and hooked it up Artisan. In principle it all works fine, we get 2 channels of data: from the Bean Temperature probe (BT) and the Infrared Bean Temperature Sensor (IBTS). Now the BT data is assigned to the  ET channel in Artisan, and the IBTS data is assigned to the BT channel in Artisan and we're wondering if that can be swapped.
All annotations are drawn on the BT channel, which means that the will appear (during roasting) along the IBTS line where we would prefer to have them along the Bean Probe line just for easier comparison with our main roaster (Giesen W6E) that has no infrared sensor.

I'm aware that after roasting session is finished I can do a "Swap BT<->ET" to get what I want afterwards, but I'm wondering if I can already have them swapped during roasting?

Thanks,
Dennis

**Comment by MAKOMO** [comment, marked as answer]
You can swap ET and BT channels by ticking the ET<->BT flag in the Curves dialog, 2nd tab "Filter" under "Input Filter"
